pages/index.tsx

```diff
@@ -85,4 +72,6 @@ const Home: NextPage = () => {
   );
 };
 
-export default Home;
+export default dynamic(() => Promise.resolve(Home), {
```

Comment:
Since we have `"use client"` on the top, do we still need to do this to disable server-side rendering?

Reply:
yes, next still tries to initially paint something on the server and it causes hydration errors